packages/vue-email/src/components/e-button.vue

```diff
@@ -1,72 +1,52 @@
 <template>
-  <a
-    v-bind="$props"
```

Comment:
Binding props is the way to pass the n element attributes. I thing that it's important to keep it.